src/window.rs

```diff
@@ -141,8 +141,11 @@ impl Window {
         window.show();
 
         // Maximize window after mapping in X11
-        if cfg!(not(any(target_os = "macos", windows))) && event_loop.is_x11() {
-            window.set_maximized(window_config.start_maximized());
+        #[cfg(not(any(target_os = "macos", windows)))]
+        {
+            if event_loop.is_x11() {
+                window.set_maximized(window_config.start_maximized());
```

Comment:
Another issue with doing it like this is that we could be potentially unsetting the WMs maximized state if a user sets the window to maximized through the WM instead of Alacritty.

So when using `set_maximized` after start, we should only do so when maximized is `true`, we shouldn't unmaximize the window when it is `false`.

So `if event_loop.is_x11() && window_config.start_maximized() { window.set_maximized(true); }` is probably what we want.